SingleViewApi/V1/Infrastructure/SingleViewContext.cs

```diff
@@ -16,8 +16,9 @@ public SingleViewContext(DbContextOptions options) : base(options)
 
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
-            modelBuilder.Entity<CustomerDbEntity>();
-            // .HasMany(x => x.CustomerDataSource);
+            modelBuilder.Entity<CustomerDbEntity>()
+                // .HasMany(x => x.DataSources)
+                ;
```

Comment:
should this ; be there?

Reply:
for this change yes. 
I might need this when I search for user